app/templates/Application/contact.html.twig

```diff
@@ -36,19 +36,19 @@
             {{ form_row(form.comment) }}
         </fieldset>
     </div>
+    <div class="row">
+        <div class="col-sm-12">
+            <input type="submit" class="btn btn-primary pull-right" value="Submit" />
+        </div>
+    </div>
     <div class="row phone">
         <fieldset class="col-sm-12">
             <p>Leave this field empty. We do not need your phone number but bots will think we do.</p>
             <p>Thanks for helping us keep this contact form bot-free!</p>
+        </fieldset>
+        <fieldset class="col-sm-12">
```

Comment:
The description and phone # were supposed to be intentionally hidden by CSS IIRC as we don't require a phone. If it's filled out then it's a bot who scraped the page. Not sure why phone is still showing up on the form.